modules/ROOT/pages/database-administration/aliases/manage-aliases-standard-databases.adoc

```diff
@@ -0,0 +1,842 @@
+----
+(no changes, no records)
+----
+
+
+
+[[alias-management-drop-database-alias]]
+== Delete database aliases
+
+////
+CREATE ALIAS garden.flowers FOR DATABASE `perennial-flowers` PROPERTIES { perennial: true };
+CREATE ALIAS `northwind-2022` FOR DATABASE `northwind-graph-2022` PROPERTIES { newestNorthwind: true, index: 3 };
+CREATE ALIAS `northwind` FOR DATABASE `northwind-graph-2021`;
+CREATE ALIAS `remote-northwind-2021`
+FOR DATABASE `northwind-graph-2021` AT 'neo4j+s://location:7687' USER alice PASSWORD 'password'
+PROPERTIES { newestNorthwind: false, index: 6 };
+CREATE ALIAS `remote-northwind` FOR DATABASE `northwind-graph-2020` AT "neo4j+s://other-location:7687" USER alice PASSWORD 'password';
+CREATE ALIAS `remote-with-driver-settings` FOR DATABASE `northwind-graph-2020` AT "neo4j+s://location:7687" USER bob PASSWORD 'newPassword'
+DRIVER {
+  connection_timeout: duration({ minutes: 1 }),
+   logging_level: "debug"
+   };
+CREATE ALIAS garden.trees FOR DATABASE updatedTrees AT 'neo4j+s://location:7687'
+USER alice PASSWORD 'password'
+PROPERTIES { treeVersion: 2 }
+////
+
+
+Both local and remote database aliases can be deleted using the `DROP ALIAS` command.
+The required privileges are described in the link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/administration/access-control/dbms-administration/#access-control-dbms-administration-alias-management[Cypher Manual -> Access control -> DBMS administration].
+
+
+=== Delete local database aliases
+
+Delete a local database alias.
+
+
+.Query
+[source, cypher]
+----
+DROP ALIAS `northwind` FOR DATABASE
+----
+
+When a database alias has been deleted, it will no longer show up in the `aliases` column provided by the command `SHOW DATABASES`.
+
+.Query
+[source, cypher]
+----
+SHOW DATABASE `northwind-graph-2021`
+----
+
+.Result
+[role="queryresult"]
+----
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| name                   | type       | aliases | access       | address          | role      | writer | requestedStatus | currentStatus | statusMessage | default | home  | constituents |
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| "northwind-graph-2021" | "standard" | []      | "read-write" | "localhost:7687" | "primary" | TRUE   | "online"        | "online"      | ""            | FALSE   | FALSE | []           |
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+----
+
+
+
+
+=== Delete remote database aliases
+
+Delete a remote database alias.
+
+.Query
+[source, cypher]
+----
+DROP ALIAS `remote-northwind` FOR DATABASE
+----
+
```

Comment:
should we have something like
```
When a database alias has been deleted, it will no longer show up in the `SHOW ALIASES FOR DATABASE` command.

.Query
[source, cypher]
----
SHOW ALIASES `remote-northwind` FOR DATABASE
----

// show empty result
```

Reply:
Is that not obvious? Or would users expect to find deleted aliases when looking for aliases?

Reply:
We have that for the drop composite alias I think it was, so before when it was just one section we had one example showing it removed from the alias column and one that showed it no longer was returned in the show alias one. But since the section is now split only one of those things are mentioned/showed 🤷 

Might be nice to have that part here and then in the composite show that it's no longer listed in the constituent column for the composite database instead?

Reply:
Have added - let me know what you think @renetapopova

Reply:
The standard alias version has been added, awaits potential update in the composite alias version